sw/device/lib/base/log.h

```diff
@@ -128,6 +134,16 @@ void base_log_internal_dv(log_severity_t severity, const char *format, ...);
  * literal.
  * @param ... format parameters matching the format string.
  */
-#define LOG_ERROR(...) LOG(kLogSeverityError, __VA_ARGS__)
+#define LOG_ERROR(...) LOG(LOG_SEVERITY_ERROR, __VA_ARGS__)
+
+/**
+ * Log a fatal error.
+ *
+ * @param severity a severity of type |log_severity_t|.
+ * @param format a format string, as described in print.h. This must be a string
+ * literal.
+ * @param ... format parameters matching the format string.
+ */
+#define LOG_FATAL(...) LOG(LOG_SEVERITY_FATAL, __VA_ARGS__)
```

Comment:
I don't want to add LOG_FATAL unless it contains an abort call. Otherwise, what even is the difference between a fatal error and a normal error?